test/conformance/walk.go

```diff
@@ -199,6 +199,11 @@ func (v *visitor) emit(arg ast.Expr) {
 			return
 		}
 
+		err := validateTestName(at.Value)
+		if err != nil {
+			v.failf(at, err.Error())
```

Comment:
I guess `return` could be necessary like line 199 after `failf()` call to make the process stop.

Reply:
done